extensions/vscode/README.md

```diff
@@ -1,11 +1,60 @@
-# Posit Publisher
+# Posit Publisher - Deploy to Posit Connect
 
-[Posit Publisher](https://github.com/posit-dev/publisher) is an extension for
-[Visual Studio Code](https://code.visualstudio.com/) that provides an easy
-way to deploy to [Posit Connect](https://posit.co/products/enterprise/connect/).
+[Posit Publisher](https://github.com/posit-dev/publisher) is a code-first tool
+for deploying to
+[Posit Connect](https://posit.co/products/enterprise/connect/).
 
 ## Installation
 
-Install Posit Publisher from the
+Install Posit Publisher by using the Install button above, or using the
+Extensions side bar in [Positron](https://github.com/posit-dev/positron) or
+[VS Code](https://code.visualstudio.com/) and searching for `posit.publisher`.
```

Comment:
Trying to be more generic about the install instructions here to account for users seeing this in Positron or VS Code.